zrml/swaps/fuzz/utils.rs

```diff
@@ -0,0 +1,201 @@
+#![allow(
+    // Mocks are only used for fuzzing and unit tests
+    clippy::integer_arithmetic
+)]
+
+use zeitgeist_primitives::constants::{
+    MaxAssets, MaxTotalWeight, MaxWeight, MinAssets, MinLiquidity, MinWeight, BASE,
+};
+
+use arbitrary::{Arbitrary, Result, Unstructured};
+
+use rand::{rngs::ThreadRng, seq::SliceRandom, Rng};
+
+use zeitgeist_primitives::types::{Asset, ScalarPosition, SerdeWrapper};
+
+use zeitgeist_primitives::{
+    traits::Swaps as SwapsTrait,
+    types::{PoolId, ScoringRule},
+};
+use zrml_swaps::mock::Swaps;
+
+pub fn construct_asset(seed: (u128, u16)) -> Asset<u128> {
+    let (seed0, seed1) = seed;
+    let module = seed0 % 4;
```

Comment:
We talked about it here https://github.com/zeitgeistpm/zeitgeist/pull/542#discussion_r854871244

Reply:
Not necessarily. I would either keep `module = seed0 % 4`, but then remove `Ztg` from the `match` expression, or use `module = seed0 % 5` and keep `Ztg`.

In the end, the pool that are being created here will never look like this in nature because they would not mix categorical/scalar/combinatorial outcome assets, so I currently don't have a strong opinion about `Ztg` being included.

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/542/commits/3ea49870bcdf5895e19d3dd6e376e1ac095c917e